src/store.c

```diff
@@ -46,14 +48,15 @@ struct rill_packed header
     uint64_t ts;
     uint64_t quant;
 
-    uint64_t keys;
     uint64_t pairs;
 
-    uint64_t vals_off;
-    uint64_t data_off;
-    uint64_t index_off;
+    uint64_t data_a_off;
+    uint64_t data_b_off;
```

Comment:
Out of curiosity, did you consider using the `uint64_t data_off[2];` version for this where accessing one of the column would be done like so: `data_off[rill_col_a]`?

Reply:
We did talk about this, but didn't consider implementing it that way. I can go ahead and take this route if you wish though.

Reply:
It's fine for now. I think the new consistent naming will go a long way to help readability.